admin/my-yoast-proxy.php

```diff
@@ -0,0 +1,26 @@
+<?php
+/**
+ * WPSEO plugin file.
+ *
+ * This file acts as a proxy. It will read external files and serves the like they are located locally.
+ *
+ * @package WPSEO\Admin
+ */
+
+switch ( filter_input( INPUT_GET, 'file', FILTER_SANITIZE_STRING ) ) {
+	case 'research-webworker':
+		$my_yoast_url = 'https://my.yoast.com/api/downloads/file/analysis-worker';
+		$my_yoast_url_content_type = 'text/javascript; charset=UTF-8';
+		break;
+}
```

Comment:
Why a switch with only one case? Is this going to be extended with more cases in the future?

Reply:
Extension is very likely :)